src/dnfile/codedindex.py

```diff
@@ -12,26 +12,55 @@
 Copyright (c) 2020-2021 MalwareFrank
 """
 
-from typing import List, Tuple
+from typing import TYPE_CHECKING, Union
 
 from .base import CodedIndex
 
+if TYPE_CHECKING:
+    from .mdtable import *
```

Comment:
I am generally not a fan of "import *".  I want to get these changes in sooner rather than later, so I may change that post-merge

Reply:
no problem. would you prefer keeping the namespace (e.g. `import .mdtable as mdtable` or `import dnfile.mdtable`) or using bare symbols (e.g. `from .mdtable import Foo, Bar, Baz, ...`)? i'll start with the former.

Reply:
fixed in https://github.com/malwarefrank/dnfile/pull/17/commits/11da6b592b70a93fc9e3189aa7e118a53baeb31b